packages/vm/src/params.ts

```diff
@@ -98,4 +99,26 @@ export const paramsVM: ParamsDict = {
     maxBlobGasPerBlock: 1179648, // The max blob gas allowable per block
     blobGasPriceUpdateFraction: 5007716, // The denominator used in the exponential when calculating a blob gas price
   },
+  /**
+   * EIP-7892
+   * Baseline schedule for BPO1: increases blob target to 10 and max to 15
+   */
+  7892: {
+    // gasConfig
+    targetBlobGasPerBlock: BPO1_BLOB_SCHEDULE.targetBlobGasPerBlock,
+    maxBlobGasPerBlock: BPO1_BLOB_SCHEDULE.maxBlobGasPerBlock,
+    blobGasPriceUpdateFraction: BPO1_BLOB_SCHEDULE.blobGasPriceUpdateFraction,
+  },
+  bpo1: {
+    // gasConfig
+    targetBlobGasPerBlock: BPO1_BLOB_SCHEDULE.targetBlobGasPerBlock,
+    maxBlobGasPerBlock: BPO1_BLOB_SCHEDULE.maxBlobGasPerBlock,
+    blobGasPriceUpdateFraction: BPO1_BLOB_SCHEDULE.blobGasPriceUpdateFraction,
+  },
```

Comment:
Can we then just hardcode the respective values here, and use `computeBpoSchedule()` just as an internal helper once a new BPO fork arrives to compute these values for the new setup? This is a static calculation if I see correctly, right? And then we have this as we always have, just the pure values in the dictionaries.

Reply:
Ok.  included the calculation in comments just for clarity